tensorrt_llm/serve/openai_server.py

```
@@ -288,6 +288,11 @@ async def create_chat_response(
                 if request.stream else chat_response_post_processor,
                 postproc_args=postproc_args,
             )
+            
+            lora_request = None
+            if hasattr(request, 'lora_request') and request.lora_request:
+              lora_request = request.lora_req
```

Ruff (0.12.2)

Local variable `lora_request` is assigned to but never used

Remove assignment to unused variable `lora_request`

(F841)